manager/manifests/operator.yaml

```diff
@@ -20,10 +20,9 @@ metadata:
 ---
 
 apiVersion: rbac.authorization.k8s.io/v1
-kind: RoleBinding
+kind: ClusterRoleBinding
 metadata:
   name: operator
-  namespace: $CORTEX_NAMESPACE
```

Comment:
Can we still run the operator in the cortex namespace? Where will it run without line 26, the default namespace?